manuscript/draft.Rmd

```diff
@@ -253,71 +252,60 @@ The site-observer model found that 70% of the variance in richness in the
-macroecological model has the best coverage, though each model still
-underestimated the possible variation in richness. The "naive" model
-was the only model whose confidence intervals were too wide (Figure 3), which
-can be attributed to the rapid rate at which these intervals expand as the time
-lag increases (Figure 1).
-
-RMSE and deviance produced qualitatively similar results in most cases, with the
-notable exception of the stacked SDMs (Figure 3A and 3B). Deviance, unlike RMSE,
-depends on the full predictive distribution of a model, including the model's
-estimated uncertainty. As a result, the stacked SDMs' deviance was notably
-higher than the next-worst model even though its RMSE was not unusual. On the
-other hand, the GBM-based "macroecological" model and especially the mistnet
-JSDM had reasonably well-calibrated uncertainty estimates, even in the presence
-of uncorrected observer biases (Figure 3C). This prevented their deviance from
-rising as quickly as the site-level models as the observer population evolved
-and resulted in better comparative performance for these models in terms of
-deviance (Figure 3B).
+(Figure 2; R^2 XX-XX%). However SDMs (both stacked and joint) and the
+macroecological model all failed to successfully forecast the highest-richness
+sites resulting in a notable clustering of predicted values near ~60 species and
+the poorest performance in the cross-site comparison (R^2 XX-XX%).
+
+In general, each model's error increased more-or-less linearly with time,
+although the slopes and initial error rates varied across models (Figure 3). The
+site-level models (i.e. the baselines and the ARIMA methods) tended to do very
+well (Figure 3), especially on short time horizons, and especially in terms of
+RMSE. Of these, the "Naive" model performed especially well at sites with
+uncorrected observer turnover (since its predictions were best-calibrated to
+observers that were active in later years).
+
+Most models produced confidence intervals that were too narrow, indicating
+overconfident predictions. The random forest-based SDM stack was the most
+overconfident model, with observed richness values frequently falling well
+outside its confidence limits (Figure 3C). This stacked SDM's narrow
+predictive distribution also caused it to have notably higher deviance (Figure
+3B) than the next-worst model, even though its point estimates were not
+unusually bad in terms of RMSE (3A), and the other models' relative performance
+was not nearly as sensitive to the choice of error metric. As discussed
+elsewhere [@harris2015], this overconfidence is a product of the assumption in
+stacked SDMs that errors in the species-level predictions are independent.
+Because they avoided this assumption, the GBM-based "macroecological" model and
+especially the mistnet JSDM had reasonably well-calibrated uncertainty
+estimates, even in the presence of uncorrected observer biases (Figure 3C). This
```

Comment:
Again, not sure why we want to talk about the uncorrected observer biases here instead of in its own paragraph.